docs/hacking/chrome-extension.rst

```diff
@@ -20,7 +20,7 @@ To build and install the Chrome extension:
 
    .. code-block:: bash
 
-      hypothesis-buildext chrome --debug --service 'http://127.0.0.1:5000'
+      hypothesis-buildext chrome --debug --service 'http://127.0.0.1:5000' --bouncer-url "http://127.0.0.1:8000/*"
```

Comment:
This is for development builds only. I don't think there's any reason not to include local bouncer support in dev builds? Sure not everyone is gonna use it, but when you do need to, it'll work. And putting this here is simpler for the docs and the developer experience than having to find a separate section somewhere else.